umbraco-cloud/upgrades/version-specific-upgrades/migrate-from-umbraco-7-to-8.md

```diff
@@ -0,0 +1,161 @@
+---
+description: This article will provide detailed steps on how to migrate a Umbraco 7 Cloud project to Umbraco 8.
+---
+
+# Migrate from Umbraco 7 to Umbraco 8 on Umbraco Cloud
+
+Taking your Umbraco CMS project from Umbraco 7 to 8 is called a migration as it requires that the data is migrated in the process. This article covers each step involved in this process.
+
+Read the [general article about Content migration](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/upgrading/version-specific/migrate-content-to-umbraco-8.md#what-are-the-limitations) to learn more about limitations and other things related to migrating your Umbraco site from 7 to 8.
```

Comment:
Link is not working (Broken link)

Reply:
That's because it's linking to an article that will be added with this PR ;)

Reply:
ah ok.. got it.. 😀